packages/e2e-tests/specs/experiments/blocks/navigation.test.js

```diff
@@ -343,12 +343,14 @@ describe( 'Navigation', () => {
 
 			await selectDropDownOption( 'Test Menu 2' );
 
-			await page.waitForSelector( '.wp-block-navigation__container' );
+			await page.waitForSelector(
+				'.interface-interface-skeleton__content .wp-block-navigation__container'
+			);
 
 			// Scope element selector to the Editor's "Content" region as otherwise it picks up on
 			// block previews.
 			const navBlockItemsLength = await page.$$eval(
-				'[aria-label="Editor content"][role="region"] div[aria-label="Block: Custom Link"]',
+				'.interface-interface-skeleton__content .wp-block-navigation__container [data-type="core/navigation-link"]',
```

Comment:
I'm not sure about this change, using aria-labels has been considered good practice on the project. Maybe we can start with the suggestion in my other comment and see how that goes.